CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -153,6 +179,11 @@ local function OnCharacterAdded(character)
 	
 	backpack.ChildAdded:connect(OnChildAdded)
 	backpack.ChildRemoved:connect(OnChildRemoved)
+
+	if humanoid then
+		humanoid.Changed:connect(function() UpdateHealth(humanoid) end)
+		UpdateHealth(humanoid)
+	end
```

Comment:
Would you still believe me if I told you I've been making games on ROBLOX since 2007?